.ci/docker/common/install_conda.sh

```diff
@@ -45,8 +45,13 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
 
   # Prevent conda from updating to 4.14.0, which causes docker build failures
   # See https://hud.pytorch.org/pytorch/pytorch/commit/754d7f05b6841e555cea5a4b2c505dd9e0baec1d
-  # Uncomment the below when resolved to track the latest conda update
-  # as_jenkins conda update -y -n base conda
+  # Uncomment the below when resolved to track the latest conda update,
+  # but this is required for CentOS stream 9 builds
+  ID=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
+  OS_VERSION=$(grep -oP '(?<=^VERSION_ID=).+' /etc/os-release | tr -d '"')
+  if [[ $ID == centos && $OS_VERSION == 9 ]]; then
+    as_jenkins conda update -y -n base conda
+  fi
```

Comment:
@janeyx99 This was commented via [your commit](https://github.com/pytorch/pytorch/commit/1eff853fdc91619ea24abf0cbd51ca992fe10c97) 3 years ago. Is it safe to unconditionally uncomment it now? :)